bin/config-ghost.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const Cp = require('child_process');
+const Consulite = require('consulite');
+const consulHost = `${process.env.CONSUL}:8500`;
+const debug = require('debug')('config-ghost');
+const isPreStart = process.argv[2] === 'preStart';
+const isOnChange = process.argv[2] === 'onChange';
+
+debug('running node script. preStart=%s, onChange=%s, env=%j',
+  isPreStart, isOnChange, process.env);
+
+if (!isPreStart && !isOnChange) {
+  throw new Error('run in an unexpected mode');
+}
+
+lookupService((err, service) => {
+  if (err) {
+    throw err;
+  }
+
+  debug('service = %j', service);
+  createGhostConfigFile(service);
```

Comment:
We just looked up the service a couple lines up. If it wasn't found, an error is thrown. Technically, the service could have gone down between then and now, but I don't think we can handle that.